### PR TITLE
Add example: turners's cube

### PR DESCRIPTION
**Pull request by mkovaxx**
Plans for a nifty turner's cube that my grandpa made on a lathe, long before I was even born.

Thought it would be cool to upload it to the digital realm. ;)

IMG_6464

### CODE REVIEW

File: examples/src/turners_cube.rs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    wire.to_face().extrude(DVec3::new(0.0, 0.0, edge_length)).into_shape()
+}
+
+fn rolling_pin(wp: &Workplane, r1: f64, h1: f64, r2: f64, h2: f64) -> Shape {
+    let c_long = cylinder(&wp, r1, h1);
+    let c_wide = cylinder(&wp, r2, h2);
+    c_long.union(&c_wide).into_shape()
+}
+
+fn cylinder(wp: &Workplane, radius: f64, height: f64) -> Shape {
```
Comment (bschwind) [inline examples/src/turners_cube.rs:57]: I need to start adding these primitives to `Workplane` or somewhere similar

**Review by bschwind — APPROVED** [review]
LGTM!

File: examples/src/turners_cube.rs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+fn cube(edge_length: f64) -> Shape {
+    let wp = Workplane::xy();
+    let wp = wp.translated(DVec3::new(0.0, 0.0, -0.5 * edge_length));
+    let wire = wp.rect(edge_length, edge_length);
+    wire.to_face().extrude(DVec3::new(0.0, 0.0, edge_length)).into_shape()
+}
+
+fn rolling_pin(wp: &Workplane, r1: f64, h1: f64, r2: f64, h2: f64) -> Shape {
+    let c_long = cylinder(&wp, r1, h1);
+    let c_wide = cylinder(&wp, r2, h2);
```
Comment (bschwind) [inline examples/src/turners_cube.rs:53]: Clippy is complaining about these two

```suggestion
    let c_long = cylinder(wp, r1, h1);
    let c_wide = cylinder(wp, r2, h2);
```

**Comment by mkovaxx** [comment]
@bschwind Whoa, thanks for fixing the lints, and apologies for being late to the party! 😅

File: examples/src/turners_cube.rs (hunk 1 above)
Comment (mkovaxx) [reply examples/src/turners_cube.rs:57]: I agree that `Workplane` is a good place for anything that needs to be parameterized by a local frame.

**Pull request merged**